picard/mbjson.py

```diff
@@ -192,6 +194,11 @@ def _translate_artist_node(node):
     if config.setting['translate_artist_names']:
         locale = config.setting["artist_locale"]
         lang = locale.split("_")[0]
+
+        if (config.setting['translate_artist_names_exception'] and
+                config.setting["artist_locale_exception"] in detect_alphabet(node['name'])):
```

Comment:
Our coding style wants the `and` to be at the beginning of the line, so this should do.

```
        if (config.setting['translate_artist_names_exception']
                and config.setting["artist_locale_exception"] in detect_alphabet(node['name'])):
```